### PR TITLE
Component Reuse Section

### PR DESCRIPTION
**Pull request by espinr**
Section that describes how components are reused in MiniApp implementations (+@MichaelWangzitao )

***
<a href="https://pr-preview.s3.amazonaws.com/espinr/miniapp-components/pull/17.html" title="Last updated on Feb 2, 2023, 9:55 AM UTC (442dd03)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/miniapp-components/17/39b4830...espinr:442dd03.html" title="Last updated on Feb 2, 2023, 9:55 AM UTC (442dd03)">Diff</a>

### CODE REVIEW

**Comment by espinr** [comment]
I've merged all the branches (one per individual section) into one to have a full picture of the document: https://raw.githack.com/espinr/miniapp-components/all-new-sections/index.html#sec-reuse

File: index.html [hunk 1]
```diff
@@ -1,1642 +1,231 @@
...
-            </td>
-          </tr>			  
-		</tbody>
-	  </table>
+  <section id="sec-reuse">
+    <h2>Component Reuse</h2>
+    <p>
+      A component may use other custom components registered and defined within the package. For that, developers must import the custom component in the parent component declaration. The imported component is exposed to the component template. It will be available as another tag in the template, identified by a unique key.
+    </p>
+    <pre class="example" title="Import of custom component"><code>&lt;!-- my-component.html -&gt;
```
Comment (xfq) [inline index.html]:
```suggestion
    <pre class="example" title="Import of custom component"><code>&lt;!-- my-component.html --&gt;

```

File: index.html [hunk 2]
```diff
@@ -1,1642 +1,231 @@
...
+&lt;import src="./cool-button" name="cool-button" /&gt;
+
+&lt;div&gt;
+  &lt;cool-button&gt;&lt;/cool-button&gt;
+&lt;/div&gt;</code></pre>
+    <p data-cite="HTML">
+      Like in Web Components, MiniApp components may use [^slot^] elements as placeholders to pass content from the parent to the child component.
+    </p>  
+    <pre class="example" title="Use of slot elements inside components"><code>&lt;!-- my-component.html -&gt;
```
Comment (xfq) [inline index.html:138]:
```suggestion
    <pre class="example" title="Use of slot elements inside components"><code>&lt;!-- my-component.html --&gt;

```

File: index.html [hunk 3]
```diff
@@ -1,1642 +1,232 @@
...
-		</tbody>
-	  </table>
+  <section id="sec-reuse">
+    <h2>Component Reuse</h2>
+    <p>
+      A component may use other custom components registered and defined within the package. For that, developers must import the custom component in the parent component declaration. The imported component is exposed to the component template. It will be available as another tag in the template, identified by a unique key.
+    </p>
+    <pre class="example" title="Import of custom component"><code>&lt;!-- my-component.html --&gt;
+
+&lt;import src="./cool-button" name="cool-button" /&gt;
```
Comment (Sharonzd) [inline index.html:130]: Is "./cool-button" a menu or an HTML? 

( Just a question, no suggestion )

**Review by Sharonzd — APPROVED** [review]

File: index.html (hunk 3 above)
Comment (espinr) [reply index.html:130]: It's an HTML. The entry of the imported component.

**Pull request merged**